recipes-omnect/initrdscripts/omnect-os-initramfs/fs-link

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+fs_link_enabled() {
+    return 0
+}
+
+run_fs_link() {
+
+    FS_LINK_FILE="/rootfs/etc/omnect/create-fs-links.txt"
```

Comment:
as far as i understand the file will persist and thus this script recreates links on every boot.

Reply:
will refactor that part to create the link just once

Reply:
done